modules/server_new/src/main/scala/observe/server/ObserveEngine.scala

```diff
@@ -660,28 +660,28 @@ object ObserveEngine {
 
     override def selectSequence(
       i:        Instrument,
-      sid:      Observation.Id,
+      obsId:    Observation.Id,
       observer: Observer,
       user:     User,
       clientId: ClientId
     ): F[Unit] =
       val author = s", by '${user.displayName}' on client ${clientId.value}."
       executeEngine.inject(
         systems.odb
-          .read(sid)
+          .read(obsId, reset = true)
```

Comment:
I guess this is the key change. maybe a comment on why it is true would be good here